commerce-product-asset-categories-web/src/main/java/com/liferay/commerce/product/asset/categories/web/internal/display/context/CategoryCPDisplayLayoutDisplayContext.java

```diff
@@ -132,17 +131,13 @@ public String getLayoutBreadcrumb(Layout layout) throws Exception {
 			sb.append(LanguageUtil.get(httpServletRequest, "public-pages"));
 		}
 
-		sb.append(StringPool.SPACE);
-		sb.append(StringPool.GREATER_THAN);
-		sb.append(StringPool.SPACE);
+		sb.append(" > ");
```

Comment:
this one.... actually could stay the way it was. We can use multiple `StringPool` strings in a row, but only if there's no literal string on either side. It's just that if we're already writing a literal string, adding one more character (or a few) to it doesn't really make a difference.